surefire-its/src/test/resources/surefire-2010-parameterized-selection-does-not-work/src/test/java/de/dagere/peass/ExampleTestJUnit4.java

```diff
@@ -0,0 +1,36 @@
+package de.dagere.peass;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
```

Comment:
It would be nice to adjust the code style in these files well.
Pls configure your IDE, here is instructions and code style XML for your IDEs:
https://maven.apache.org/developers/conventions/code.html#intellij-idea

Reply:
Thank, I've imported the style and hopefully reformatted it correctly.